components/TextFields/src/ContainedInputView/private/MDCBaseTextFieldLayout.m

```diff
@@ -117,4 +140,8 @@ - (BOOL)displaysSideView:(UIView *)subview
   return displaysSideView;
 }
 
+- (CGFloat)textHeightWithFont:(UIFont *)font {
+  return (CGFloat)ceil((double)font.lineHeight);
+}
+
```

Comment:
I just noticed that no unit tests were added for this layout logic added in the last PR; please add unit tests for the layout logic (this is a great object to be tested independently).

Reply:
I added some initial unit tests for the layout object. I can add more as time goes by and things begin to settle 👍